hwdb.d/60-sensor.hwdb

```diff
@@ -1075,6 +1076,7 @@ sensor:modalias:acpi:KIOX010A*:dmi:*:svnTREKSTOR:pnYOURBOOKC11B:*
 
 sensor:modalias:acpi:KIOX020A*:dmi:*:svnTREKSTOR:pnPrimebookC11B:*
 sensor:modalias:acpi:KIOX020A*:dmi:*:svnTREKSTOR:pnPRIMEBOOKC11B:*
+sensor:modalias:acpi:KIOX010A*:dmi:*:svnTREKSTOR:pnPrimebookC13:*
```

Comment:
duplicated, this is the same as the other line being added

Reply:
thanks, should be fixed now. I missed changing the second lines ACPI to KIOX020A*